.golangci.yml

```diff
@@ -1,52 +1,66 @@
-linters-settings:
-  gocyclo:
-    min-complexity: 25
-  errcheck:
-    check-type-assertions: true
-  forbidigo:
-    forbid:
-      - '^fmt\.Print'
-      - '^log\.'
-      - '^print$'
-      - '^println$'
-      - '^panic$'
-  godox:
-    # TODO, OPT, etc. comments are fine to commit. Use FIXME comments for
-    # temporary hacks, and use godox to prevent committing them.
-    keywords: [FIXME]
-  varnamelen:
-    ignore-decls:
-      - ok bool
-      - T any
-      - i int
-      - wg sync.WaitGroup
+version: "2"
 linters:
-  enable-all: true
+  default: all
   disable:
     - cyclop            # covered by gocyclo
     - depguard
     - exhaustive
     - exhaustruct
-    - exportloopref     # deprecated since v1.60.2
     - funlen            # rely on code review to limit function length
     - gochecknoglobals  # globals are fine
     - gocognit          # dubious "cognitive overhead" quantification
-    - gofumpt           # prefer standard gofmt
-    - goimports         # rely on gci instead
-    - mnd               # some unnamed constants are okay
     - ireturn           # "accept interfaces, return structs" isn't ironclad
     - lll               # don't want hard limits for line length
     - maintidx          # covered by gocyclo
+    - mnd               # some unnamed constants are okay
     - nilnil
     - nlreturn          # generous whitespace violates house style
-    - protogetter
```

Comment:
Went ahead and enabled this - there were initially issues with early versions but they seem resolved.